### PR TITLE
device-plugins.md: add generic-device-plugin

### PR DESCRIPTION
**Pull request by squat**
I'd like to add a reference to a project I created several years ago called https://github.com/squat/generic-device-plugin, which allows Kubernetes to discover and allocate generic Linux devices such as the FUSE device, serial devices, Linux video devices, sound cards, USB devices, etc. This project provides access to Linux devices that do not require special, vendor-specific setup logic. It is often used in conjunction with Iot deployments, e.g. to expose ZigBee devices to a home assistant Deployment.

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -470,6 +470,7 @@ Here are some examples of device plugin implementations:
 * The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
 * The [SR-IOV Network device plugin](https://github.com/intel/sriov-network-device-plugin)
 * The [Xilinx FPGA device plugins](https://github.com/Xilinx/FPGA_as_a_Service/tree/master/k8s-device-plugin) for Xilinx FPGA devices
+* The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices and USB devices
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:473]: Please preserve the existing collation order (items are sorted alphabetically)

**Review by sftim — COMMENTED** [review]
Thanks for the PR

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -460,6 +460,7 @@ pluginapi.Device{ID: "25102017", Health: pluginapi.Healthy, Topology:&pluginapi.
 Here are some examples of device plugin implementations:
 
 * The [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
+* The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices and USB devices
```
Comment (squat) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:463]: Is this good now , @sftim? I'm ignoring casing here for sorting

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md (hunk 1 above)
Comment (squat) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: 👍 done

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm
/approve

**Comment by squat** [comment]
Oddly the jobs haven't finished running; it looks like Netlify's pipeline isn't updating its state: the build says it finished but the next step isn't running. Do we need to retrigger CI?

**Comment by ydFu** [comment]
Currently, there is an issue with the Netlify CI tool.

I would suggest using the following command to trigger a Netlify rebuild, which has the advantage of not losing the lgtm label and reducing the duplication of code views:

```shell
git reset
git commit --amend --no-edit
git push --force-with-lease
```

**Pull request merged**